File: app/src/main/java/com/byoutline/kickmaterial/views/DisabledSeekBar.java
package com.byoutline.kickmaterial.views;

import android.content.Context;
import android.util.AttributeSet;
import android.view.MotionEvent;
import android.widget.SeekBar;

public class DisabledSeekBar extends SeekBar {

    public DisabledSeekBar(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
    }

    public DisabledSeekBar(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public DisabledSeekBar(Context context) {
        super(context);
    }

    @Override
    public boolean onTouchEvent(MotionEvent event) {
        return true;
    }
}

File: app/src/main/java/com/byoutline/kickmaterial/views/DisabledSeekBar.kt
package com.byoutline.kickmaterial.views

import android.content.Context
import android.support.v7.widget.AppCompatSeekBar
import android.util.AttributeSet
import android.view.MotionEvent

class DisabledSeekBar : AppCompatSeekBar {

    constructor(context: Context, attrs: AttributeSet, defStyle: Int) : super(context, attrs, defStyle)

    constructor(context: Context, attrs: AttributeSet) : super(context, attrs)

    constructor(context: Context) : super(context)

    override fun onTouchEvent(event: MotionEvent): Boolean = true
}
